app/controllers/restaurants_controller.rb

```diff
@@ -0,0 +1,58 @@
+class RestaurantsController < ApplicationController
```

Comment:
🔨 @sbwheeler let's get this indentation issue sorted out, can you re-install sublime 3?